students/HarveyHerela/lesson02/part2.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+# Prints a specific row
+# Use 'edge' as the corner pieces and middle
+# and 'filler' as the filler pieces
+def print_row( edge, filler, side ):

```

Comment:
Python/PEP8 style for function declarations is one space between parameters, no space between parantheses and parameters, e.g. def print_row(edge, filler, side):